src/ranger/ranger_resource_policy_manager.cpp

```diff
@@ -206,7 +208,12 @@ ranger_resource_policy_manager::ranger_resource_policy_manager(
 
 void ranger_resource_policy_manager::start()
 {
-    CHECK_NOTNULL(_meta_svc, "");
+    if (_meta_svc == nullptr) {
+        // `_meta_svc` being null implies that the policies will be updated manually,
```

Comment:
Is the situation only possible in tests?

Reply:
Yes, it's only used for test. I'll add `MOCK_TEST` macro to control it. Do not use `NDEBUG` since tests running on Github are built with `Release` rather than `Debug`.